composer/trainer/checkpoint.py

```diff
@@ -1,5 +1,7 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""Load and save checkpoints during training."""
```

Comment:
Thoughts about making this module private? I think I saw this discussed on slack; would be in favor of that. We'll just need to add a helper on the trainer class to get the saved checkpoints.